dbfit-java/oracle/src/main/java/dbfit/environment/OracleEnvironment.java

```diff
@@ -344,13 +345,24 @@ private DbParameterAccessor makeSingleParam(
             paramDirection = getParameterDirection(direction);
         }
 
-        DbParameterAccessor dbp = new OracleDbParameterAccessor(paramName,
+        DbParameterAccessor dbp = createOracleDbParameterAcccessor(
+                paramName,
                 paramDirection, getSqlType(dataType), getJavaClass(dataType),
                 paramPosition, normaliseTypeName(dataType), userTypeName);
 
         return dbp;
     }
 
+    private OracleDbParameterAccessor createOracleDbParameterAcccessor(
```

Comment:
I can rename/overload but not add the annotation as we're not overriding the super-type's method.